goal_src/engine/ambient/mood-tables.gc

```diff
@@ -5,3 +5,4106 @@
 ;; name in dgo: mood-tables
 ;; dgos: GAME, ENGINE
 
+;; definition for function make-light-kit
+(defun
+  make-light-kit
+  ((arg0 light-group) (arg1 float) (arg2 float) (arg3 float) (arg4 float))
+  (let ((s4-0 (new 'stack-no-clear 'matrix)))
+   (matrix-rotate-y! s4-0 arg1)
+   (let ((v1-0 (-> arg0 dir0)))
+    (set! (-> v1-0 direction x) (l.f L87))
```

Comment:
You'll need to add these labels to label_types.json, until I get around to fixing https://github.com/water111/jak-project/issues/461.  These are just floats.